R/tlf_ae_specific.R

```diff
@@ -25,7 +25,7 @@
 #' @inheritParams r2rtf::rtf_page
 #' @inheritParams r2rtf::rtf_body
 #' @param footnotes A character vector of table footnotes.
-#' @param title A character vector of table titles.
+#' @param title Term "analysis", "observation"and "population") for collecting title from metadata or a character vector of table titles.
```

Comment:
you might want to separate the argument into two `title_order` and `title`. 

Each one is for different purpose. Then you can allow a vector of user defined `title` as @BrianLang suggest.

Reply:
would it be:
- if user gives an argument for `titles`, then use those titles (and ignore `title_order`). 
- if `titles == NULL` then use `title_collect` and `title_order`. 
  - with argument checking for `title_order` of something like `sort(title_order) == sort(c("analysis", "observation", "population"))`? Or is it not necessary to have all three ?

Reply:
Exactly! The reason to have all three it to ensure flexibility in metalite. The title component can be any order of combination to fit use cases in different tables.